source/_assets/less/main.less

```diff
@@ -1,31 +1,75 @@
-@import "base";
-@import "colors";
-@import "typography";
-@import "z-index";
-@import "grid";
-@import "flex";
-@import "images";
-
-// Components
-@import "forms";
-@import "panel";
-@import "list-group";
-@import "buttons";
-@import "navbar";
-@import "svg-icon";
-@import "nav-list";
-@import "documentation-page";
-@import "code-editor";
-@import "docsearch";
-@import "files";
-@import "logo";
-
-// Utilities
-@import "backgrounds";
-@import "layout";
-@import "spacing";
-@import "borders";
-@import "utilities";
-@import "responsive-utilities";
-@import "transitions";
-@import "animations";
+@tailwind preflight;
+@tailwind components;
+@tailwind utilities;
+
+@import 'typography';
+@import 'files';
+
+// backgrounds
+.bg-gradient-purple {
+    background: #884992;
+    background: -webkit-linear-gradient(to right, #703e84, #884992);
+    background: linear-gradient(to right, #703e84, #884992);
+}
+
+.bg-opaque-purple {
+    background: rgba(45, 15, 52, 0.5);
+}
+
+// code editor
```

Comment:
These should (1) go in a separate file, and (2) be loaded between `@tailwind components` and `@tailwind utilities` so we could still override them inline, if necessary. (`@import 'typography'` should move there, too.

(according to the ordering instructions [here](https://tailwindcss.com/docs/installation/#3-use-tailwind-in-your-css) )

Reply:
Good catch, I'll restructure the `main.less` file in #34 . The code editor styles have been extracted to it's [own file](https://github.com/tightenco/jigsaw-site/blob/8004d5e7004e0bb782e388eccf783226ada96d19/source/_assets/less/code-editor.less).